src/components/Button/Button.js

```diff
@@ -2,7 +2,15 @@ import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import PropType from 'prop-types';
 
-function Button({ classList, buttonType, text, handleData }) {
+function Button(
+  /* prettier-ignore */
+  {
```

Comment:
- [ ] Dear @Rysth, Just as mentioned by the previous reviewer, please Consider removing disabled liters. Remember linter is here to force you to write clean code and help you identify what could possibly be improved on the current codebase. Please consider removing all the disabled liters.